Source/Samples/Samples.csproj

```diff
@@ -25,4 +25,21 @@
     <ProjectReference Include="..\Libs\GtkSharp\GtkSharp.csproj" />
     <ProjectReference Include="..\Libs\PangoSharp\PangoSharp.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Update="Sections\Category.cs">
+      <LogicalName>GtkSharp.Samples.Category.cs</LogicalName>
+    </EmbeddedResource>
+    <EmbeddedResource Update="Sections\ListSection.cs">
+      <LogicalName>GtkSharp.Samples.ListSection.cs</LogicalName>
+    </EmbeddedResource>
+    <EmbeddedResource Update="Sections\SectionAttribute.cs">
+      <LogicalName>GtkSharp.Samples.SectionAttribute.cs</LogicalName>
+    </EmbeddedResource>
+    <EmbeddedResource Update="Sections\Dialogs\AboutDialogSection.cs">
+      <LogicalName>GtkSharp.Samples.AboutDialogSection.cs</LogicalName>
+    </EmbeddedResource>
+    <EmbeddedResource Update="Sections\Widgets\ButtonSection.cs">
+      <LogicalName>GtkSharp.Samples.ButtonSection.cs</LogicalName>
+    </EmbeddedResource>
+  </ItemGroup>
```

Comment:
I'm guessing VS for Mac added these few lines, delete them, there is already a wildcard for them.